docs/prod-host-setup.md

```diff
@@ -64,10 +64,14 @@ nosmt=force
 Verification can be done by running:
 
 ```bash
-(grep -q "^forceoff$\|^notsupported$" /sys/devices/system/cpu/smt/control && echo "Hyperthreading: DISABLED (OK)") || echo "Hyperthreading: ENABLED (Recommendation: DISABLED)"
+(grep -q "^forceoff$\|^notsupported$" /sys/devices/system/cpu/smt/control && \
+echo "Hyperthreading: DISABLED (OK)") || \
+echo "Hyperthreading: ENABLED (Recommendation: DISABLED)"
 ```
 
-#### [Intel only] Check Kernel Page-Table Isolation (KPTI) support
+**Note** AWS Graviton processors do not have SMT.
```

Comment:
Are there aarch64 processors with SMT? If not I suggest to put at the beginning of the section brackets only for Intel and AMD